--- origsrc/kdiffdate.pro	2006-02-24 00:56:40.000000000 -0600
+++ src/kdiffdate.pro	2008-05-19 22:12:22.250000000 -0500
@@ -5,8 +5,8 @@
 HEADERS += mainwindow.h mainwidget.h
 HEADERS += version.h
 
-INCLUDEPATH += $(KDEDIR)/include
-LIBS += -lkdeui -L$(KDEDIR)/lib
+INCLUDEPATH += /usr/include/kde
+LIBS += -lkdeui -lkdecore
 
 DISTFILES += configure README INSTALL COPYING AUTHOR kdiffdate-1.0.lsm
 
